File: src/main/java/me/diax/diax/data/db/GuildData.java
package me.diax.diax.data.db;

import lombok.Data;
import me.diax.diax.data.db.extra.GuildProfile;
import me.diax.diax.data.db.extra.GuildSettings;

@Data
public class GuildData {
    public String id;
    public GuildSettings settings = new GuildSettings();
    public GuildProfile profile = new GuildProfile();
}

File: src/main/kotlin/me/diax/diax/data/db/GuildData.kt
package me.diax.diax.data.db

import me.diax.diax.data.db.extra.GuildProfile
import me.diax.diax.data.db.extra.GuildSettings

class GuildData {
    var id: String? = null
    var settings: GuildSettings = GuildSettings()
    var profile: GuildProfile = GuildProfile()

    override fun toString(): String = "GuildData(id=$id, settings=$settings, profile=$profile)"
}
